src/uct/ib/base/ib_iface.c

```diff
@@ -693,7 +682,7 @@ uct_ib_iface_roce_is_reachable(const uct_ib_device_gid_info_t *local_gid_info,
     status = ucs_sockaddr_inet_addr_size(local_ib_addr_af, &addr_size);
     if (status != UCS_OK) {
         ucs_error("failed to detect RoCE address size");
-        uct_iface_populate_info_str_buf(
+        UCT_IFACE_FILL_INFO_STR_BUF(
```

Comment:
do we also need the ucs_error above?

Reply:
No.. unless we want to provide something to uct users..?

Reply:
so let's remove line 684 `ucs_error("failed to detect RoCE address size");`